.travis.yml

```diff
@@ -5,29 +5,36 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.npm
+
+env:
+  global:
+    - NPM_CONFIG_PROGRESS='false'
 
 matrix:
   include:
     - php: '7.0'
       env: coverage=1
     - php: '7.0'
-      env: deps="low"
+      env: deps='low'
 
 before_install:
+  - phpenv config-rm xdebug.ini
   - phpunit --self-update
   - npm install -g swagger-cli
   - if [[ $coverage = 1 ]]; then mkdir -p build/logs build/cov; fi
   - if [[ $coverage = 1 ]]; then wget https://phar.phpunit.de/phpcov.phar; fi
   - if [[ $coverage = 1 ]]; then wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar; fi
 
 install:
-  - if [[ ! $deps ]]; then composer update --no-progress --ansi; fi
-  - if [[ $deps = low ]]; then composer update --no-progress --ansi --prefer-lowest --prefer-stable; fi
+  - if [[ $coverage = 1 ]]; then composer require --dev --no-update 'phpunit/php-code-coverage:^4.0.1'; fi
```

Comment:
This is not a new dependency. We're using it in CoverageContext. It was used through a transitive dependency, which is of course something we shouldn't do. (I really hope we have a dependency analysis tool to help us catch such problems!)

Shall we add it to require-dev in composer.json proper?